--- config/crowdsec.service-orig	2021-06-29 09:35:15.945984145 +0200
+++ config/crowdsec.service	2021-06-29 09:35:45.338024681 +0200
@@ -5,9 +5,9 @@
 [Service]
 Type=notify
 Environment=LC_ALL=C LANG=C
-ExecStartPre=/usr/local/bin/crowdsec -c /etc/crowdsec/config.yaml -t
-ExecStart=/usr/local/bin/crowdsec -c /etc/crowdsec/config.yaml
+ExecStartPre=/usr//bin/crowdsec -c /etc/crowdsec/config.yaml -t
+ExecStart=/usr/bin/crowdsec -c /etc/crowdsec/config.yaml
 #ExecStartPost=/bin/sleep 0.1
 ExecReload=/bin/kill -HUP $MAINPID
